Assets/_BForBoss/_Entities/Scripts/DummyTargetBehaviour.cs

```diff
@@ -24,6 +24,8 @@ protected override void Awake()
             base.Awake();
             _animator = GetComponentInChildren<Animator>();
             _renderer = GetComponentInChildren<Renderer>();
+            if(_healthbar != null) 
+                _healthbar.Initialize(_lifeCycle);
```

Comment:
You can probably do a null check on validate, since it's a `SerializeField`

Reply:
Like throw a warning there?

Reply:
yeah